from exceptions import Exception

class DatosInvalidos(Exception):
    pass

